packages/nextjs/src/common/devErrorSymbolicationEventProcessor.ts

```diff
@@ -26,13 +31,20 @@ async function resolveStackFrame(
       params.append(key, (frame[key as keyof typeof frame] ?? '').toString());
     });
 
+    let basePath = globalWithInjectedValues.__sentryBasePath ?? '';
+
+    // Prefix the basepath with a slash if it doesn't have one
+    if (basePath !== '' && !basePath.match(/^\//)) {
```

Comment:
l: can we do a `startsWith('/')` check instead of the regex?

Reply:
I would but it's not IE11 :(